cabot/metricsapp/api/elastic_grafana.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import json
+from elasticsearch_dsl import Search, A
+from elasticsearch_dsl.query import Range
+from django.core.exceptions import ValidationError
+from cabot.metricsapp.defs import ES_SUPPORTED_METRICS
+
+
+logger = logging.getLogger(__name__)
+
+
+def build_query(series, min_time='now-10m', default_interval='1m'):
+    """
+    Given series information from the Grafana API, build an Elasticsearch query
+    :param series: a "target" in the Grafana dashboard API response
```

Comment:
hmm I could do this here or when I initially get them from the Grafana API response? Which would be best?

Might prefer when I initially get them since it'll identify an error earlier